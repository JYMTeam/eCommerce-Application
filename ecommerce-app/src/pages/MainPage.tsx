import { Box, Container } from "@mui/material";
import React from "react";
import { MainPageCarousel } from "../components/MainPageCarousel/MainPageCarousel";
import MainPageBanner from "../components/MainPageBanner";
export function MainPage() {
  return (
    <Container>
      <Box mt={6} mb={2}>
        <MainPageCarousel />
        <MainPageBanner />
      </Box>
    </Container>
  );
}
