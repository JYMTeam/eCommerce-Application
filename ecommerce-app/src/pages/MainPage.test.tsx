import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { MainPage } from "./MainPage";

describe("MainPage component", () => {
  it("renders main page heading", () => {
    render(<MainPage />);
    const headingElement = screen.getByText("Main Page");
    expect(headingElement).toBeInTheDocument();
  });

  it("matches snapshot", () => {
    const { container } = render(<MainPage />);
    expect(container).toMatchSnapshot();
  });
});
