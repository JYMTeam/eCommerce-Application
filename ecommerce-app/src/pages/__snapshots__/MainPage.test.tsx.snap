// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component matches snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <h2
      class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter css-47suzj-MuiTypography-root"
    >
      Main Page
    </h2>
  </div>
</div>
`;
